blackarch-mirrorlist

```diff
@@ -92,6 +92,9 @@ Server = https://ftp.halifax.rwth-aachen.de/blackarch/$repo/os/$arch
 #Server = http://mirror.yandex.ru/mirrors/blackarch/$repo/os/$arch
 #Server = ftp://mirror.yandex.ru/mirrors/blackarch/$repo/os/$arch
 
+#South Korea
```

Comment:
```suggestion
# South Korea
```

Reply:
I you can just fix the country spacing so it's like for the others. Outside of this LGTM.

Reply:
oke did